ehr/api-src/org/labkey/api/ehr/SharedEHRUpgradeCode.java

```diff
@@ -120,6 +125,42 @@ else if (methodName.startsWith(IMPORT_FROM_TSV_PREFIX))
 
             _tsvImports.add(new TsvImport(schemaName, queryName, tsvPath));
         }
+        else if (methodName.startsWith(IMPORT_DOMAIN_TEMPLATE))
+        {
+            String[] tsvArguments = methodName.split(";");
+            if (tsvArguments.length != 3)
+            {
+                throw new UnsupportedOperationException("Expected three arguments for importTemplate but got " + (tsvArguments.length - 1));
```

Comment:
why -1 here?

Reply:
I guess the first one is importTemplate itself?

Reply:
Yeah that is how the error message for importFromTsv counts the params so I was just being consistent with that.

Reply:
I'd recommend at least renaming the variable here - this isn't the TSV case

Reply:
done